II.pass-tests/ES-6-practice/es6.js

```diff
@@ -1,17 +1,25 @@
 // Turn this function into an arrow function
-function arrow() {
-  return "es6 is awesome!";
-}
+//  function arrow() {
+//   return "es6 is awesome!";
+// }
+const arrow= () => {
+  return "es6 is awesome!";}
+
 
 //Use the es6 syntax for default parameter
-function defaultParameter(name) {
-  var name = name || "sam";
-  return name;
-}
+// function defaultParameter(name) {
+//   var name = name || "sam";
+//   return name;
+// }
+  function defaultParameter(name = "sam") {
+    return name;
+  }
+
 
 // Use the spread operator to combine arr1 and arr2
-function combineArrays(arr1, arr2) {
-  return arr1.concat(arr2);
+// 
+  function combineArrays(arr1, arr2) {
+    return [...arr1, ...arr2];
 }
 
 //use destructuring to return the object's cyf property
```

Comment:
You are doing really well with ES6 :tada: 
It seems that you forgot to change `destructing` and `templateString`
Keep on going :muscle:

Reply:
yes I forgot